pkg/runtime/compstore/compstore.go

```diff
@@ -48,7 +49,7 @@ type ComponentStore struct {
 	pubSubs                 map[string]PubsubItem
 	topicRoutes             map[string]TopicRoutes
 	workflowComponents      map[string]workflows.Workflow
-	workflowBackends        map[string]wfbe.WorkflowBackend
+	workflowBackends        map[string]backend.Backend
```

Comment:
Why this change?

Reply:
This is the correct type for the workflow backend implementation interface. Notice the other component types are using the interface types defined in `components-contrib` however backends is using `durabletask-go` directly. This wasn't picked up before as this field wasn't previously being used (`workflowbackend.go` didn't exist). `wfbe.WorkflowBackend` is a factory func type.